public/faq.md

```diff
@@ -1 +1,100 @@
+
+- __What if I have a question not answered in this FAQ?__
+  
+  Please make sure to read the website,manual. If after reading through it you still don't have an answer please consider reaching out to us on admin@kossiitkgp.org.
+
+
+
+# STUDENT
+- __How should I get selected in KWOC?__
+
+  There is absolutely no selection criteria for students. You are selected in the program as soon as you register. So you don't have to ask your mentors "How to get selected for your project?", but instead go ahead with **"How can I contribute to your project?"**.
+
+- __My mentor is not replying, What should I do?__
+
+  Please be patient with your mentor(s). If you do not get a response on the communication channel, contact the mentor on their email id. If no response for 3 days, inform us.
+
+- __What is the last date of student registration?__
+
+  Students registration deadline is 6th December. The contribution chart goes live at the same time, so be quick. 
+  > early bird catches the worm :) .
+
+- __Can I show the contributions that I have made in KWoC for participating in any competitions?__
+  
+  All the contributions that you will make in KWoC will be displayed on your GitHub timeline. So, that is a big plus if you are participating in any competitions and your GitHub profile is on display.
+
+- __Can I apply if I am not a student of IIT Kharagpur?__
+
+  Sure ! Our program is open to __all__ students irrespective of college ! Is there any eligibility criteria and pre-requisites? There is no strict eligibility criteria as such. Read [this](https://www.quora.com/What-is-the-prerequisite-for-participating-in-Kharagpur-Winter-of-Code-2016) Quora answer by Pranit Bauva.
+
+- __Can you provide me any advice so that I can start preparing for it beforehand?__
+  
+  Play around git and GitHub. Create an account on GitHub, set up git on your machine and start experimenting with it. You can also begin by following Codecademy's git interactive tutorial.
+
+- __What are my chances of receiving a position in KWoC?__
+  Pretty fair ! You need to be working for the whole month regardless of your expertise level.
+
+- __Is there any other similar kind of program that I might find interesting?__
+
+  Other programs similar to KWoC are [GirlScript Summer of Code](https://gssoc.girlscript.tech) organised by GirlScript India. If you meant similar to GSoC, then check out [this](https://github.com/deepanshu1422/List-Of-Open-Source-Internships-Programs) awesome list of programs.
+
+- __How much time in a day do I need to give for KWoC?__
+  
+  GSoC requires 40 hours per week in summers. For KWoC, you may give 1-2 hours daily and that includes conversation with your mentor(s). More the projects you contribute to, more the time in conversing about the work.
+
+- __Will I receive a certificate at the end of the KWoC?__
+  
+  If you remain with us till the end of the program, then you'll receive a certificate for completion of the program.
```

Comment:
Verbose answer here would be good. 
```suggestion
  If you remain with us till the end of the program, then you'll receive a certificate for completion of the program. You would have to clear both the mid and end evaluation to get the certificate. More details can be found in our [manual](link here)
```

Reply:
need to edit student,mentor manuals of 2k20,i will edit and ask for u r reviewms and will put link here.ok? @sahil-shubham

Reply:
edit means changing 2020 to 2021 mainly,will also read them and will suggest something,if something is to be improvised

Reply:
Sounds good. Add a TODO here to update the link, just in case so that we don't forget.